src/db/models/planVersion.ts

```diff
@@ -10,3 +14,43 @@ export type PlanVersionId = Brand<
 >;
 
 export const PLAN_VERSION_ID = brandedType<number, PlanVersionId>(t.number);
+
+const PLAN_VERSION_CLUSTER_SELECTION_TYPE = {
+  single: null,
+  multi: null,
+};
+
+export default defineLegacyVersionedModel({
+  tableName: 'planVersion',
+  fields: {
+    generated: {
+      id: { kind: 'branded-integer', brand: PLAN_VERSION_ID },
+    },
+    nonNullWithDefault: {
+      isForHPCProjects: { kind: 'checked', type: t.boolean },
+    },
+    accidentallyOptional: {
+      planId: { kind: 'branded-integer', brand: PLAN_ID },
+      name: { kind: 'checked', type: t.string },
+      startDate: { kind: 'checked', type: DATE },
+      endDate: { kind: 'checked', type: DATE },
+    },
+    optional: {
+      comments: { kind: 'checked', type: t.string },
+      code: { kind: 'checked', type: t.string },
+      customLocationCode: { kind: 'checked', type: t.string },
+      currentReportingPeriodId: {
+        kind: 'branded-integer',
+        brand: PLAN_REPORTING_PERIOD_ID,
+      },
+      lastPublishedReportingPeriodId: { kind: 'checked', type: t.number },
+      // Even though this column isn't defined using DB enum only two values are used
+      clusterSelectionType: {
+        kind: 'enum',
+        values: PLAN_VERSION_CLUSTER_SELECTION_TYPE,
+      },
```

Comment:
should use a checked t.keyof type instead

Reply:
Should have removed this comment above too. Since it "justifies" why `'enum'` was used